simulation_study/plot-case-study-denisties.R

```diff
@@ -0,0 +1,180 @@
+###########################
+# Function definitions ----
+###########################
+
+source("helper_functions.R") # for GPD density and quantiles
+
+density_discontinuous <- function(x, uniform_lower, uniform_upper, proportion_uniform, gpd_scale, gpd_shape){
+
+  is_undefined <- which(x < uniform_lower)
+  is_uniform <- which((x > uniform_lower) & (x < uniform_upper))
+  is_gpd <- which(x > uniform_upper)
+
+  density <- rep(NA_real_, length(x))
+  density[is_undefined] <- 0
+  density[is_uniform] <- proportion_uniform / (uniform_upper - uniform_lower)
+  density[is_gpd] <- dgpd(x[is_gpd], gpd_shape, gpd_scale, mu = uniform_upper)
+  density[is_gpd] <- density[is_gpd] * (1 - proportion_uniform)
+  return(density)
+}
+
+density_continuous <- function(x, alpha = 1, beta = 0.5){
+  gpd_scale = 0.5
+  gpd_shape = 0.1
+  
+  prop_to_density <- function(x, a = alpha, b = beta){
+    dgpd(x, shape = 0.1, scale = 0.5) * pbeta(x, a, b)
+  } 
+
+  # Area correction to get valid pdf
+    AUC_0_to_1 <- integrate(prop_to_density, 0, 1)$value
+    AUC_1_to_inf <- 1 - pgpd(q = 1,shape = gpd_shape, scale = gpd_scale)
```

Comment:
I think AUC_1_to_inf should be using the adjusted GPD parameters conditioned on being above 1, i.e., scale=0.6?

Reply:
Have added additonal plots comparing sample CDF to analytical CDFs using mine and Jon's expression.